scripts/build_helpers/bin/build_dart.dart

```diff
@@ -47,8 +66,21 @@ void main(List<String> args) async {
       exit(-1);
     }
 
-    if (!await _buildDart()) {
-      exit(-1);
+    if (buildType == "all") {
```

Comment:
I think there's a simpler way to do this. I'll take a stab at it and modify the PR this evening.

Reply:
Here's my stab at supporting multiple targets: https://github.com/fuzzybinary/dart_shared_libray/blob/987261ef05376b1d1dea0dbd4bbcab23ae5c9f86/scripts/build_helpers/bin/build_dart.dart#L12